tests/test_tutorial/test_handling_errors/test_tutorial004.py

```diff
@@ -8,18 +9,20 @@
 def test_get_validation_error():
     response = client.get("/items/foo")
     assert response.status_code == 400, response.text
-    # TODO: remove when deprecating Pydantic v1
-    assert (
-        # TODO: remove when deprecating Pydantic v1
-        "path -> item_id" in response.text
-        or "'loc': ('path', 'item_id')" in response.text
-    )
-    assert (
-        # TODO: remove when deprecating Pydantic v1
-        "value is not a valid integer" in response.text
-        or "Input should be a valid integer, unable to parse string as an integer"
-        in response.text
-    )
+    if PYDANTIC_VERSION_MINOR_TUPLE < (2, 0):
+        assert (
+            response.text
+            == """1 validation error
+path -> item_id
+  value is not a valid integer (type=type_error.integer)"""
+        )
+    else:
+        assert (
+            response.text
+            == """1 validation error
+path -> item_id
+  Input should be a valid integer, unable to parse string as an integer (type=int_parsing)"""
+        )
```

Comment:
```suggestion
        assert response.text == (
            "1 validation error\n"
            "path -> item_id\n"
            "  Input should be a valid integer, unable to parse string as an integer "
            "(type=int_parsing)"
        )
```
I think this is more readable